graph_snyk_cve_sync.py

```diff
@@ -127,6 +139,10 @@ def _parse_data_for_eco(self, eco):
         """Return True/False depending on whether the ecosysytem data should be parsed or not."""
         if self.helper.is_delta_mode_on():
             return True
+        if eco == self.selective_eco_run:
+            return True
+        elif self.selective_eco_run != "false":
+            return False
```

Comment:
no its not. if we dont give this check, then it will not only run for the ecosystem you have asked for, but also for the ecosystem that is supposed to run on that particular day

Reply:
default value is false. i can make is none if false doesnt sound good

Reply:
Then it is better to rename this to `none` instead of `false`.

Reply:
ok. noted